modules/insights-operator-configuring.adoc

```diff
@@ -5,23 +5,23 @@
 
 :_mod-docs-content-type: CONCEPT
 [id="insights-operator-configuring_{context}"]
-= Configuring Insights Operator
+= Configuring {insights-operator}
```

Comment:
Below we say "Using the {insights-operator}", but here we say "Configuring {insights-operator}".